src/core/Notification/Notification.md

```diff
@@ -18,15 +18,14 @@ import {
     This is a small screen error notification text.
   </Notification>
   <Notification
-    id="notification-1"
+    id="notification-id"
     closeText="Close"
     headingText="Notification heading"
     headingVariant="h3"
+    closeButtonProps={{ 'aria-label': 'Custom close button label' }}
```

Comment:
Not a 100% sure this is something we want to emphasize in the example. It was added to test adding `aria-label` through the `closeButtonProps` and decided to leave it there for now.